src/components/article/index.js

```diff
@@ -95,7 +99,7 @@ class ArticleFullPage extends Component {
                     {key !== 0 && section.image ? <div className='articleImage' style={{backgroundImage: `url(${section.image})`}}><img
                         src={section.image}/></div> : null}
                     {section.title ? (key === 0 ? <h1>{section.title}</h1> : <h2 >{section.title}</h2>) : null}
-                    {section.text ? <p className='articleText'>{section.text}</p> : null}
+                    {section.text ? <div className='articleText' dangerouslySetInnerHTML={this.rawMarkup(section.text)}/> : null}
```

Comment:
this method always cracks me up! 
https://facebook.github.io/react/tips/dangerously-set-inner-html.html